IAMOTZ/ievents
# #153236209 Implement the sign in page with react

#### What does this PR do?
It uses react and redux to implement the sign-in page consuming the API endpoint for logging in users.
#### Description of Task to be completed?
User should be able to signin on the application
#### How should this be manually tested?
- clone the repo
- cd into the project directory
- Run `npm install` to install all the dependency
- Run `npm run react:dev`
- visit the port where the webpack-dev-server started, the signup page should be loaded there
- If the login was successful, the page should be redirected to another sample page where you see all your details from the application
- If the login was not successful, the error should be displayed right at the top of the form
#### Any background context you want to provide?
- The frontend still uses the locally hosted server, so you would have to create a .env file to fulfill all the needed environment variables, the run `npm run server:dev`
#### What are the relevant pivotal tracker stories?
#153236209
#### Screenshots (if appropriate)
![screencapture-localhost-8080-users-login-1512040206807](https://user-images.githubusercontent.com/26048536/33427981-755ebdec-d5c7-11e7-9ed8-1046895f63eb.png)
#### Questions:
None
